lib/src/main/java/com/auth0/jwt/JWTVerifier.java

```diff
@@ -61,11 +61,11 @@ static Verification init(Algorithm algorithm) throws IllegalArgumentException {
         /**
          * Require a specific Issuer ("iss") claim.
          *
-         * @param issuer the required Issuer value
+         * @param issuer the required Issuer value. If multiple values are given, the claim must at least match one of them
          * @return this same Verification instance.
          */
         @Override
-        public Verification withIssuer(String issuer) {
+        public Verification withIssuer(String... issuer) {
             requireClaim(PublicClaims.ISSUER, issuer);
```

Comment:
Let's use this instead 

```suggestion
             requireClaim(PublicClaims.ISSUER, issuer == null ? null : Arrays.asList(issuer));
```